.github/workflows/ci.yml

```diff
@@ -319,6 +319,95 @@ jobs:
           secret_vault_address: ${{ secrets.VAULT_ADDR }}
           secret_vault_roleId: ${{ secrets.VAULT_ROLE_ID }}
 
+  opensearch-integration-tests:
+    if: needs.detect-changes.outputs.java-code-changes == 'true'
+    needs: [ detect-changes ]
+    name: "[IT] OpenSearch"
+    timeout-minutes: 10
+    permissions: {}  # GITHUB_TOKEN unused in this job
+    runs-on: gcp-perf-core-8-default
+    env:
+      ZEEBE_TEST_DOCKER_IMAGE: localhost:5000/camunda/zeebe:current-test
+    services:
+      # Set up an OpenSearch service container
+      # https://docs.github.com/en/actions/use-cases-and-examples/using-containerized-services/about-service-containers
+      opensearch:
+        image: opensearchproject/opensearch:2.18.0
```

Comment:
❓ Question and a bit of a note2self: what did we conclude last time for Elasticsearch on whether Renovate will detect/bump these dependencies... does it work or not or unknown 🤔

Reply:
Looking at https://github.com/camunda/camunda/issues/12605 I see that Renovate has _some_ "Update docker.elastic.co/elasticsearch/elasticsearch Docker tag to v8.17.1 (main)" in the backlog which _could_ match also this location: https://github.com/camunda/camunda/blame/327d53167ba114898165f1561b74c9c20649f9ff/.github/workflows/ci.yml#L243C41-L243C41

Reply:
Looking at https://github.com/camunda/camunda/pull/26179/files confirms that it should work here since it is already working on other GHA workflows.